tests/test_data_base.py

```diff
@@ -0,0 +1,77 @@
+import psycopg2
+import pytest
+from psycopg2 import sql
+
+from src.data_base import DataBase
+
+
+@pytest.fixture
+def table_name():
+    return "test_data_base"
+
+
+@pytest.fixture
+def db(table_name):
+    database = DataBase(table_name)
+    yield database
+    database.connection.close()
+
+
+@pytest.fixture
+def test_ids():
+    return "42", "43", "44"
+
+
+def test_db_context_manager(table_name):
+    with DataBase(table_name) as db:
+        assert db.table_name == table_name
+        assert isinstance(db.connection, psycopg2.extensions.connection)
+        assert isinstance(db.cursor, psycopg2.extensions.cursor)
+        assert db.cursor.closed is False
```

Comment:
assert not db.cursor.closed
Comparing to False or 0 is tying your code to internal psycopg2 implementation

Reply:
done